### ISSUE TITLE
Bundled data objects are not documented

### ISSUE THREAD

**Issue by Kdreval** [morinlab/GAMBLR#172]
```
❯ checking for missing documentation entries ... WARNING
  Undocumented code objects:
    ‘chromosome_arms_grch37’ ‘chromosome_arms_hg38’
    ‘dhitsig_genes_with_weights’ ‘gene_blacklist’ ‘grande_maf’
    ‘grch37_all_gene_coordinates’ ‘grch37_ashm_regions’
    ‘grch37_gene_coordinates’ ‘grch37_lymphoma_genes_bed’
    ‘grch37_oncogene’ ‘grch37_partners’ ‘hg38_ashm_regions’
    ‘hg38_gene_coordinates’ ‘hg38_lymphoma_genes_bed’ ‘hg38_oncogene’
    ‘hg38_partners’ ‘hotspot_regions_grch37’ ‘hotspot_regions_hg38’
    ‘lymphoma_genes’ ‘lymphoma_genes_comprehensive’ ‘reddy_genes’
    ‘target_regions_grch37’ ‘target_regions_hg38’
    ‘wright_genes_with_weights’
  Undocumented data sets:
    ‘chromosome_arms_grch37’ ‘chromosome_arms_hg38’
    ‘dhitsig_genes_with_weights’ ‘gene_blacklist’ ‘grande_maf’
    ‘grch37_all_gene_coordinates’ ‘grch37_ashm_regions’
    ‘grch37_gene_coordinates’ ‘grch37_lymphoma_genes_bed’
    ‘grch37_oncogene’ ‘grch37_partners’ ‘hg38_ashm_regions’
    ‘hg38_gene_coordinates’ ‘hg38_lymphoma_genes_bed’ ‘hg38_oncogene’
    ‘hg38_partners’ ‘hotspot_regions_grch37’ ‘hotspot_regions_hg38’
    ‘lymphoma_genes’ ‘lymphoma_genes_comprehensive’ ‘reddy_genes’
    ‘target_regions_grch37’ ‘target_regions_hg38’
    ‘wright_genes_with_weights’
  All user-level objects in a package should have documentation entries.
  See chapter ‘Writing R documentation files’ in the ‘Writing R
  Extensions’ manual.
```

**Comment by mattssca** [comment]
I'd be happy to take on this as well.

**Comment by Kdreval** [comment]
Resolved in #176 and #177
